README.md

```diff
@@ -39,6 +39,38 @@ in [documementation](https://www.tarantool.io/en/doc/latest/book/box/data_model/
 Call of function `ddl.set_schema(schema)` creates a space `_ddl_sharding_key` with two
 fields: `space_name` with type `string` and `sharding_key` with type `array`.
 
+Similarly for `sharding_func`: call of function `ddl.set_schema(schema)` creates
+a space `_ddl_sharding_func` with three fields: `space_name`, `sharding_func_name` and
+`sharding_func_body` with type string.
+
+If you want to use sharding function from some module, you need to require the module
+with sharding function first. For example: to use sharding functions like
+[vshard.router.bucket_id_strcrc32](https://www.tarantool.io/doc/latest/reference/reference_rock/vshard/vshard_api/#router-api-bucket-id-strcrc32)
+and
+[vshard.router.bucket_id_mpcrc32](https://www.tarantool.io/doc/latest/reference/reference_rock/vshard/vshard_api/#router-api-bucket-id-mpcrc32)
+from `vshard` module you need to require `vshard` module.
+
+Also, you can pass your own sharding function by defining the function name in `_G`
+or by specifying lua code directly in `body` field:
+```lua
+sharding_func = {
+  body = 'function(key) return <...> end'
+}
+```
+
+Your defined sharding function in `_G` should have type `function`
+or `table` | `cdata` | `userdata` with `__call` method.
+
+Defined function must have a prototype according to the following rules:
+
+**Parameters**
+
+key (number | table | string) – a sharding key.
```

Comment:
I guess yes, because `vshard` sharding functions accept any Lua object as key as described in [docs](https://www.tarantool.io/en/doc/latest/reference/reference_rock/vshard/vshard_api/#router-api-bucket-id-strcrc32)

Reply:
So technically we should add `boolean` into the list of Lua types.

Reply:
Updated:

```diff
-key (number | table | string) – a sharding key.
+key (number | table | string | boolean) – a sharding key.
```